Package.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version: 5.8
+
+import PackageDescription
+
+private let packageName = "SimpleNetworkService"
+private let networkService = "NetworkService"
+private let networkServiceTest = networkService + "Tests"
+
+let package = Package(
+    name: packageName,
+    platforms: [
+        .iOS(.v15)
+    ],
+    products: [
+        .library(
+            name: networkService,
```

Comment:
nit:

Is the name of the result library "NetworkService" or "SimpleNetworkService"?

Reply:
I looked at the examples of packages on GitHub and on the apple's documentation, really the name should be "SimpleNetworkService" for the library.